src/main/scala/com/gu/fastly/api/FastlyApiClient.scala

```diff
@@ -234,7 +235,7 @@ case class FastlyApiClient(apiKey: String, serviceId: String, config: Option[Asy
     private lazy val defaultConfig = new DefaultAsyncHttpClientConfig.Builder()
       .setMaxConnections(50)
       .setMaxRequestRetry(3)
-      .setRequestTimeout(20000)
+      .setRequestTimeout(Duration.ofMillis(20000))
```

Comment:
The signature of `setRequestTimeout` has changed to receive a `Duration`. It was previously `int` that was getting converted to milliseconds: https://github.com/AsyncHttpClient/async-http-client/commit/0ebfd642b4b1c83d3c14febebb935a4f7e6729d9